packages/perseus/src/widgets/expression/expression.test.tsx

```diff
@@ -277,12 +278,10 @@ describe("Expression Widget", function () {
 
         it("should return undefined if rubric.value is null/undefined", () => {
             // Arrange
-            const rubric = {
+            const rubric: PerseusExpressionRubric = {
```

Comment:
Is there a need to explicitly type `rubric` here?  Doesn't the call to `getOneCorrectAnswerFromRubric()` type check the input value?

Reply:
If `rubric` doesn't have a type declaration, TS will infer the types of `answerForms` and `functions` to be `never[]`. That prevents them from being passed to `getOneCorrectAnswerFromRubric`. Previously `as const` was used to make the types `readonly never[]`, which is assignable to any readonly array type. But since `getOneCorrectAnswerFromRubric` now requires read-write arrays, that's no longer an option.